sos/report/__init__.py

```diff
@@ -1453,9 +1453,16 @@ def final_work(self):
             if not archive:
                 print("Creating archive tarball failed.")
             else:
-                # compute and store the archive checksum
-                hash_name = self.policy.get_preferred_hash_name()
-                checksum = self._create_checksum(archive, hash_name)
+                try:
+                    # compute and store the archive checksum
+                    hash_name = self.policy.get_preferred_hash_name()
+                    checksum = self._create_checksum(archive, hash_name)
+                except Exception:
+                    print("")
+                    print(_("Error while accessing temp dir specified. "
+                            "Please check that the directory exists."))
+                    print("")
```

Comment:
You can use `\n` in the single print statement with the message to generate the newlines, e.g.

`print(_("\nError while [...] exists.\n"))`

We could probably make this a bit more specific. `Error generating archive checksum after archive creation.` I think gives us a bit more insight here.

Reply:
ACK to both `\n` and message content.

Apart of that, +1 for the PR. Thanks!